### DISCUSSION TITLE
Work with Tailwind official Prettier plugin

### DISCUSSION THREAD

**Discussion by fabriciopirini** [sannajammeh/tw-classed#74] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not really, it would just be cool if we could add that to the workflow with this library too.

**Describe the solution you'd like**
Once I run Prettier, it would automatically sort the tailwind classes in the way it makes more sense and establish a certain pattern throughout the codebase.

**Describe alternatives you've considered**
So far, I've been doing it manually by creating a html element in the code and copying-pasting the classes to it, letting Prettier fix the order and copying-pasting it back to the `classed` call.

**Additional context**
Here is the plugin page: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier 
and its repo: https://github.com/tailwindlabs/prettier-plugin-tailwindcss

Not sure how to even start this since it seems like it is very "tied" to the [HTML structure and the class attributes](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/01fe07a29aeefcc7933e5cec22a840ab07fd3918/tests/test.js#L64) but maybe we could figure something out.

**Comment by sannajammeh** [comment]
Thanks for the suggestion. Looked through the source code of the plug-in, it only activates on `class` or `className`, so this wouldn't be possible here I'm afraid. 

I recommend taking advantage of the multi argument support for sorting components. Although its more manual. 

See this guide: https://tw-classed.vercel.app/docs/guide/writing-clean-classed#organizing-your-classed-components

Moving this to discussions
